logfire/_internal/config.py

```diff
@@ -332,6 +336,12 @@ class _LogfireConfigData:
     scrubbing_callback: ScrubCallback | None
     """A function that is called for each match found by the scrubber."""
 
+    inspect_arguments: bool
+    """Whether to enable f-string magic"""
+
+    tail_sampling: TailSamplingOptions | None
+    """Tail sampling options"""
```

Comment:
Could just be a typeddict?

Reply:
I tried that, I've also wanted `ConsoleOptions` and `PydanticPlugin` to be typed dicts, but they seemed worse. They can't define defaults in the class, and passing in a plain dict is actually not that user friendly.

Reply:
Makes sense